packages/specification/prescriptions-api.yaml

```diff
@@ -0,0 +1,269 @@
+      description: |
+        An [OAuth 2.0 bearer token](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#user-restricted-apis).
+      required: true
+      schema:
+        type: string
+        format: '^Bearer\ [[:ascii:]]+$'
+        example: "Bearer g1112R_ccQ1Ebbb4gtHBP1aaaNM"
+    NhsNumber:
+      in: header
+      name: NHS-Number
+      description: The NHS number of the patient whose prescriptions are to be returned.
+      required: true
+      schema:
+        type: string
+        example: "9434765919"
+    RequestID:
+      in: header
+      name: X-Request-ID
+      required: true
+      description: |
+        A globally unique identifier (GUID) for the request, which we use to de-duplicate repeated requests and to trace the request if you contact our helpdesk.
+        Must be a universally unique identifier (UUID) (ideally version 4).
+        Mirrored back in a response header.
+        If you re-send a failed request, use the same value in this header.
+      schema:
+        type: string
+        pattern: "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$"
+        example: 60E0B220-8136-4CA5-AE46-1D97EF59D068
+  schemas:
+    operation-outcome:
+      $ref: schemas/resources/OperationOutcome.yaml
+    bundle-response:
+      type: object
+      required:
+        - entry
+        - resourceType
+        - type
+      description: A FHIR searchset Bundle.
+      properties:
+        resourceType:
+          type: string
+          description: FHIR resource type.
+          default: Bundle
+        id:
+          type: string
+          description: Object ID for the Bundle.
+          example: 164996aa-29eb-4f14-a5ac-4b556a7baf6e
+        meta:
+          type: object
+          required: [lastUpdated]
+          properties:
+            lastUpdated:
+              type: string
+              description: The start date of a prescription. Helps the patient in knowing when the prescription is valid from.
+              format: date-time
+              example: "2023-06-05T10:11:12+00:00"
+        type:
+          type: string
+          description: Denotes that the bundle is a list of resources returned as a result of a search.
+          enum: [searchset]
+        entry:
+          type: array
+          description: A collection of resources contained within the Bundle.
+          items:
+            type: object
+            required:
+              - resourceType
+              - identifier
+              - type
+              - entry
+            description: A FHIR collection Bundle.
+            properties:
+              resourceType:
+                type: string
+                description: FHIR resource type.
+                default: Bundle
+              id:
+                type: string
+                description: Object ID for the Bundle.
+                example: 164996aa-29eb-4f14-a5ac-4b556a7baf6e
+              identifier:
```

Comment:
Does this serve any purpose? Do we use the supplied XRequest-ID for this, and if so, given it's synchronous does that add anything?

Reply:
Removed identifier.